### PR TITLE
added section for contamination (NFR feature extraction)

### PR DESCRIPTION
**Pull request by petefitz**

### CODE REVIEW

File: CardinalRules.md [hunk 1]
```diff
@@ -115,6 +115,14 @@ There is a human aspect to code reviews. Nobody wants to hear what they're doing
 
 Code reviewers are allowed to comment on stuff they like!.
 
+#### Feature 'Contamination'
+
+Your feature branch should normally only cover a specific functional piece of work, usually driven by user requirements. Changes that are not directly related to the feature you are working on should **not** be committed to this branch. These should instead be done as a separate feature. This means the code reviewer can concentrate on the specific realm of that features functionality, and is not distracted by having to look at unrelated committed work.
```
Comment (tim-crowther) [inline CardinalRules.md]: "The feature branch ..." is better.

**Comment by tim-crowther** [comment]
Some comments added.

**Comment by westonized** [comment]
LGTM

**Comment by stevemchugh** [comment]
LGTM

**Pull request merged**